spring-cloud-deployer-cloudfoundry/pom.xml

```diff
@@ -18,7 +18,7 @@
 		<cloudfoundry-java-lib.version>4.16.0.RELEASE</cloudfoundry-java-lib.version>
 		<pivotal-cf-client-reactor.version>2.2.0.RELEASE</pivotal-cf-client-reactor.version>
 		<commons-compress.version>1.26.0</commons-compress.version>
-		<okhttp.version>3.14.9</okhttp.version>
+		<okhttp.version>4.12.0</okhttp.version>
```

Comment:
Wrt "Should we bump major from 3.x to 4.x?"...

On the surface, the [4.x changelog](https://square.github.io/okhttp/changelogs/changelog_4x/) looks scary. However, the reality is that they have spent a lot of time and energy (their words) on keeping 4.x compatible w/ 3.x ([details](https://square.github.io/okhttp/changelogs/upgrading_to_okhttp_4/#upgrading-to-okhttp-4).

I am much less concerned of the risk at this point. However, what is the reward for doing this? CVE patch?